arangod/Aql/ConditionFinder.cpp

```diff
@@ -162,6 +162,10 @@ bool ConditionFinder::before(ExecutionNode* en) {
               // if the enumerate collection node had the counting flag
               // set, we can copy it over to the index node as well
               idx->copyCountFlag(node);
+              if (node->canReadOwnWrites() == ReadOwnWrites::yes) {
+                // copy over the read-own-writes flag from EnumerateCollectionNode to IndexNode
+                idx->setCanReadOwnWrites(ReadOwnWrites::yes);
+              }
```

Comment:
Can't we simplify like this?
```suggestion
              // copy over the read-own-writes flag from EnumerateCollectionNode to IndexNode
              idx->setCanReadOwnWrites(node->canReadOwnWrites());
```

Reply:
Yes!